src/jit/importer.cpp

```diff
@@ -6675,6 +6703,12 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
             call = impIntrinsic(newobjThis, clsHnd, methHnd, sig, pResolvedToken->token, readonlyCall,
                                 (canTailCall && (tailCall != 0)), &intrinsicID);
 
+            if (compInlineResult->IsFailure())
```

Comment:
It is because now we call impLookupToTree inside impIntrinsic, that can set Failure if it is compIsForInlining() and we need runtime lookup.